### PR TITLE
Feat: Create Merkle Tree

### PR DESCRIPTION
**Pull request by damiramirez**
**This PR:**

- Create a Merkle Tree from an input
- Add test

image

### CODE REVIEW

File: src/tree.rs [hunk 1]
```diff
@@ -0,0 +1,98 @@
+use sha3::{Digest, Sha3_256};
+
+type MerkleTree = Vec<Vec<[u8; 32]>>;
```
Comment (lima-limon-inc) [inline src/tree.rs:3]: Really cool use of Type!

**Review by dsocolobsky — APPROVED** [review]

**Review by LeanSerra — APPROVED** [review]
LGTM

**Review by lima-limon-inc — APPROVED** [review]
LGTM ^_^

**Comment by lima-limon-inc** [comment]
Should this branch be deleted? Considering that is has been successfully merged?

**Pull request merged**